js/script.js

```diff
@@ -11,83 +11,83 @@
 //When user inputs text into textbox, clicking enter will submit it
 var inputByEnter = document.getElementById("textbox");
```

Comment:
Nice job here. By creating the reference here, you only need to traverse the DOM once.